DOC-3151: The `mceInsertContent` command could delete the parent block element when an anchor was selected. (#3723)
tinymce/tinymce-docs: modules/ROOT/pages/7.9.0-release-notes.adoc

{productname} {release-version} also includes the following bug fix<es>:

=== The `mceInsertContent` command could delete the parent block element when an anchor was selected.
// #TINY-11953

The {productname} `mceInsertContent` command could inadvertently remove the parent block element when an anchor (`<a>`) tag was selected.

> In senarios such as when an anchor element was fully selected, the surrounding block-level parent tag, such as `<p>` or `<div>`, could be incorrectly removed, causing unexpected content structure changes. This issue primarily affected Chrome users, as it did not reproduce consistently in Safari or Firefox.

Reviewer: ```suggestion
In scenarios where an anchor element was fully selected, the surrounding block-level parent tag, such as `<p>` or `<div>`, could be incorrectly removed, leading to unexpected changes in the content structure. This issue primarily affected Chrome users and did not reproduce consistently in Safari or Firefox.
```

> The issue has been resolved in {release-version}. The `mceInsertContent` command now properly retains the parent block elements when inserting content into a selected anchor element.

Reviewer: ```suggestion
This issue has been resolved in {release-version}. The `mceInsertContent` command now properly retains the parent block elements when inserting content into a selected anchor element.
```
